overlord/configstate/configcore/netplan.go

```diff
@@ -0,0 +1,117 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2021 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package configcore
+
+import (
+	"fmt"
+	"strings"
+
+	// TODO: Move to yaml.v3 everywhere, there is PR#10696 that starts
+	//       this. However it is not trivial yaml.v2 accepts duplicated
```

Comment:
gci doesn't support long comments in imports, this is annoying. In theory now that we have switched to 1.13 we could switch back to use goimports, there's probably things to change because gci and goimports have slightly different expectations

Reply:
I think for now I just move the comment above the import statement.